plugins/lookup/passwordstore.py

```diff
@@ -77,6 +79,24 @@
           - warn
           - empty
           - create
+      lock:
+        description: How to synchronize operations.
+        ini:
+          - section: passwordstore_lookup
+            key: lock
+        type: str
+        default: write
+        choices:
+          - readwrite
+          - write
+          - none
+      locktimeout:
+        description: Used with C(lock!='none'), lock timeout in seconds.
+        ini:
+          - section: passwordstore_lookup
+            key: locktimeout
+        type: integer
+        default: 900
```

Comment:
Is 900 seconds (15 minutes) a good choice? It seems excessively long to me.

Reply:
That is based on personal experience:
- Run playbook
- Switch desktops
- A few minutes later switch back, realize that pinentry is waiting for a passphrase to be entered, as gpg-agent removed the key from the cache

Happened frequently enough to decide to use a pretty long timeout.

Reply:
Makes sense. And since this is configurable, if someone dislikes the long timeout (or thinks it is too short :D ) can change it.